src/Compilers/Test/Utilities/CSharp/RequiredMemberAttributesVisitor.cs

```diff
@@ -25,6 +26,47 @@ private RequiredMemberAttributesVisitor(StringBuilder builder) : base(builder)
 
     protected override SymbolDisplayFormat DisplayFormat => SymbolDisplayFormat.TestFormat;
 
+    protected override void ReportSymbol(Symbol symbol)
+    {
+        EntityHandle handle;
+        PEModule module;
+
+        switch (symbol)
+        {
+            case PENamedTypeSymbol namedType:
+                handle = namedType.Handle;
+                module = ((PEModuleSymbol)namedType.ContainingModule).Module;
+                break;
+
+            case PEFieldSymbol field:
+                handle = field.Handle;
+                module = ((PEModuleSymbol)field.ContainingModule).Module;
+                break;
+
+            case PEPropertySymbol property:
+                handle = property.Handle;
+                module = ((PEModuleSymbol)property.ContainingModule).Module;
+                break;
+
+            default:
+                base.ReportSymbol(symbol);
+                return;
+        }
+
+        var attribute = module.GetAttributeHandle(handle, AttributeDescription.RequiredMemberAttribute);
```

Comment:
>var attribute = module.GetAttributeHandle(handle, AttributeDescription.RequiredMemberAttribute); [](http://example.com/codeflow?start=8&length=96)

We probably want to call ```base.ReportSymbol(symbol);``` for this code path too, to verify that attributes filtered out as expected.